packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -128,6 +129,11 @@ export function getCommonConfig(wco: WebpackConfigOptions) {
     });
   }
 
+  if (buildOptions.autoBundleWorkerModules) {
+    const workerPluginInstance = new WorkerPlugin();
```

Comment:
As per my reply to googlechromelabs/worker-plugin#20, you can now pass an option to suppress the `globalObject` warning:

```suggestion
    const workerPluginInstance = new WorkerPlugin({
      globalObject: false
    });
```

Reply:
Following our conversation in https://github.com/GoogleChromeLabs/worker-plugin/issues/20, I think `false` really is the best option for our case since we don't emphasize HMR in Angular CLI. We'll probably revisit this setting in the future.

Reply:
Yup! I figured that'd be the case. Cheers.